runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -2985,18 +2985,32 @@ static void init_amHandling(void);
 
 static
 void init_ofiForAms(void) {
-  //
-  // Compute the amount of space we should allow for AM landing zones.
-  // We should have enough that we needn't re-post the multi-receive
-  // buffer more often than, say, every tenth of a second.  We know from
-  // the Chapel performance/comm/low-level/many-to-one test that the
-  // comm=ugni AM handler can handle just over 150k "fast" AM requests
-  // in 0.1 sec.  Assuming an average AM request size of 256 bytes, a 40
-  // MiB buffer is enough to give us the desired 0.1 sec lifetime before
-  // it needs renewing.  We actually then split this in half and create
-  // 2 half-sized buffers (see below), so reflect that here also.
-  //
-  const size_t amLZSize = ((size_t) 40 << 20) / 2;
+
+  // Specify the amount of space we should allocate for AM landing zones. We
+  // should have enough that we needn't re-post the multi-receive buffer more
+  // often than, say, every tenth of a second. The original value was 40MB as
+  // explained in the comment below. The default is now 64MB and it's
+  // configurable, although more experiments are necessary to perform a
+  // similar size computation on more modern hardware.
+  //
+  // Original comment:
+  // We know from the Chapel performance/comm/low-level/many-to-one test that
+  // the comm=ugni AM handler can handle just over 150k "fast" AM requests in
+  // 0.1 sec.  Assuming an average AM request size of 256 bytes, a 40 MiB
+  // buffer is enough to give us the desired 0.1 sec lifetime before it needs
+  // renewing.  We actually then split this in half and create 2 half-sized
+  // buffers (see below), so reflect that here also.
+  //
+  const char *ev = "COMM_OFI_LZ_SIZE";
+  const size_t defaultSize = (size_t) 64 << 20;
+  const char *sizeStr = chpl_env_rt_get(ev, "64M");
```

Comment:
This confused me having two defaults. Can we get rid of the `64M` default here and just let the sizeStr default be `NULL`? `

Reply:
I can do that.